src/main/java/org/gridsuite/study/server/StudyService.java

```diff
@@ -1595,8 +1595,18 @@ public Mono<Void> updateEquipmentCreation(UUID studyUuid, String createEquipment
             Mono<Void> monoUpdateStatusResult = updateStatuses(studyUuid, nodeUuid, false);
 
             return networkModificationService.updateEquipmentCreation(createEquipmentAttributes, modificationType, modificationUuid)
-                .doOnSuccess(e -> networkModificationTreeService.notifyModificationNodeChanged(studyUuid, nodeUuid))
-                .then(monoUpdateStatusResult);
+                    .doOnSuccess(e -> networkModificationTreeService.notifyModificationNodeChanged(studyUuid, nodeUuid))
+                    .then(monoUpdateStatusResult);
+        });
+    }
+
+    public Mono<Void> updateEquipmentModification(UUID studyUuid, String modifyEquipmentAttributes, ModificationType modificationType, UUID nodeUuid, UUID modificationUuid) {
+        return Mono.zip(getModificationGroupUuid(nodeUuid), getVariantId(nodeUuid)).flatMap(tuple -> {
```

Comment:
The tuple seems unused, questionnable. If that's right, a comment would be welcome.